vault/resource_kv_secret_v2_test.go

```diff
@@ -55,17 +60,23 @@ func testKVSecretV2Config(mount, name string) string {
 
 	ret += fmt.Sprintf(`
 resource "vault_kv_secret_v2" "test" {
-  mount                      = vault_mount.kvv2.path
-  name                       = "%s"
-  cas                        = 1
-  delete_all_versions        = true
-  data_json                  = jsonencode(
-  {
-    zip       = "zap",
-    foo       = "bar",
-    flag      = false
-  }
+  mount               = vault_mount.kvv2.path
+  name                = "%s"
+  delete_all_versions = true
+  data_json = jsonencode(
+    {
+      zip  = "zap",
+      foo  = "bar",
+      flag = false
+    }
   )
+  custom_metadata {
```

Comment:
nit: might be worth preserving the original test config as well i.e. testing when without and without the `custom_metadata` being set.

Reply:
Good call, added an extra test step